### PR TITLE
php-fpm integration

### PR DESCRIPTION
**Pull request by altuner**

### CODE REVIEW

File: collector/php-fpm/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+# Ingest metrics using the PHP-FPM integration
+
+The OTel Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
+
+Please note that not all metrics receivers available for the OpenTelemetry Collector have been tested by Lightstep Observability, and there may be bugs or unexpected issues in using these contributed receivers with Lightstep Observability metrics. File any issues with the appropriate OpenTelemetry community.
```
Comment (carlosalberto) [inline collector/php-fpm/README.md:6]: This is the first time I see such a notice - is this the case or have we used this one already for other components @nslaughter @altuner ?

File: collector/php-fpm/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+You can run this example with `docker-compose up` in this directory.
+
+## Configuration
+
+Installation of the OpenTelemetry Collector varies, please refer to the [collector documentation](https://opentelemetry.io/docs/collector/) for more information.
+
+PHP-FPM integration requires enabling status page and the use of [PHP-FPM Exporter](https://github.com/hipages/php-fpm_exporter).
+
+PHP-FPM metrics described [here](https://github.com/hipages/php-fpm_exporter#metrics-collected).
```
Comment (carlosalberto) [inline collector/php-fpm/README.md:23]: *are described

**Comment by carlosalberto** [comment]
In general LGTM - the Prometheus exporter seems just fine, specially as it has Docker support, which hopefully will simplify usage. We may need to extend it later with an example using the CLI instead (that will depend on CS feedback from users).

(Next time we need additional components to run Prometheus export, let's have a brief conversation, just be to be double sure, as in general we want to avoid complications, etc)

File: collector/php-fpm/README.md (hunk 1 above)
Comment (nslaughter) [reply collector/php-fpm/README.md:6]: We've used this in quite a few. 
We should get more consistent, but it's boilerplate that we have used frequently.

**Review by nslaughter — APPROVED** [review]
Alright. I head in mind that when I had commented I left approval too.

**Pull request merged**